docs/components/compact.md

```diff
@@ -246,7 +246,7 @@ The main and only `Service Level Indicator` for Compactor is how fast it can cop
 
 To understand that you can use mix `thanos_objstore_bucket_last_successful_upload_time` being quite fresh, `thanos_compactor_halted` being non 1 and `thanos_blocks_meta_synced{state="loaded"}` constantly increasing over days.
 
-![Example view of compactor not coping with amount and size of incoming blocks](compactor_no_coping_with_load.png)
+<img src="./compactor_no_coping_with_load.png" class="img-fluid" alt="Example view of compactor not coping with amount and size of incoming blocks"/>
```

Comment:
What's the reasoning for switching to pure HTML vs using Markdown syntax?

Reply:
@squat most of the image used in doc directory are using a class called **img-fluid** which is used to make the image size proper and responsive according to the device but the image that was used in compact.md was not being styled. so that's why it was taking the entire screen.

Reply:
@squat  thanks for the review, I have updated the pull request, can you take a look again? :)

Reply:
thanks @Itaachime :)

Reply:
Ci is to be failing and seems to want this to be simplified to:
```suggestion
<img src="compactor_no_coping_with_load.png" class="img-fluid" alt="Example view of compactor not coping with amount and size of incoming blocks"/>
```